src/test/utils.ts

```diff
@@ -0,0 +1,13 @@
+import * as vscode from 'vscode'
+
+export function resetExtensionState(extension: vscode.ExtensionContext, secrets: string[] = []) {
```

Comment:
There is a similar function in `PostHogProvider.ts`, called `resetState`, only used in dev mode for debugging purposes.